### PR TITLE
CLOUDP-196976: Publish compliancePolicy commands

### PR DESCRIPTION
**Pull request by blva**
## Proposed changes

_Jira ticket:_ [CLOUDP-196976](https://jira.mongodb.org/browse/CLOUDP-196976)

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by gssbzn — APPROVED** [review]

File: docs/atlascli/command/atlas-backups-compliancePolicy-policies-ondemand-update.txt [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+     - false
+     - Output format. Valid values are json, json-path, go-template, or go-template-file. To see full output, use the -o json option.
+   * - --projectId
+     - string
+     - false
+     - Hexadecimal string that identifies the project to use. This option overrides the settings in the configuration file or environment variable.
+   * - --retentionUnit
+     - string
+     - true
+     - Unit of time in which MongoDB Cloud measures snapshot retention: 'days' 'weeks' 'months'.
```
Comment (kanchana-mongodb) [inline docs/atlascli/command/atlas-backups-compliancePolicy-policies-ondemand-update.txt:53]:
```suggestion
     - Unit of time in which Atlas measures snapshot retention: 'days' 'weeks' 'months'. 
```

File: docs/atlascli/command/atlas-backups-compliancePolicy-policies-ondemand-update.txt [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+     - false
+     - Hexadecimal string that identifies the project to use. This option overrides the settings in the configuration file or environment variable.
+   * - --retentionUnit
+     - string
+     - true
+     - Unit of time in which MongoDB Cloud measures snapshot retention: 'days' 'weeks' 'months'. 
+   * - --retentionValue
+     - int
+     - true
+     - Duration in days, weeks, or months that MongoDB Cloud retains the snapshot. For less frequent policy items, MongoDB Cloud requires that you specify a value greater than or equal to the value specified for more frequent policy items.
```
Comment (kanchana-mongodb) [inline docs/atlascli/command/atlas-backups-compliancePolicy-policies-ondemand-update.txt:57]:
```suggestion
     - Duration in days, weeks, or months that Atlas retains the snapshot. For less frequent policy items, Atlas requires that you specify a value greater than or equal to the value specified for more frequent policy items.
```
Could you please replace MongoDB Cloud with Atlas throughout this PR?

File: docs/atlascli/command/atlas-backups-compliancePolicy-encryptionAtRest.txt [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Manage Encryption at Rest of the backup compliance policy for your project. Encryption at rest enforces all clusters with a Backup Compliance Policy to use Customer Key Management.
```
[the remark this answers is not in the document]
Comment (blva) [reply docs/atlascli/command/atlas-backups-compliancePolicy-encryptionAtRest.txt]: thanks, updated!

File: docs/atlascli/command/atlas-backups-compliancePolicy-policies-ondemand-create.txt [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+     - false
+     - Hexadecimal string that identifies the project to use. This option overrides the settings in the configuration file or environment variable.
+   * - --retentionUnit
+     - string
+     - true
+     - Unit of time in which MongoDB Cloud measures snapshot retention: 'days' 'weeks' 'months'. 
+   * - --retentionValue
+     - int
+     - true
+     - Duration in days, weeks, or months that MongoDB Cloud retains the snapshot. For less frequent policy items, MongoDB Cloud requires that you specify a value greater than or equal to the value specified for more frequent policy items.
```
[the remark this answers is not in the document]
Comment (blva) [reply docs/atlascli/command/atlas-backups-compliancePolicy-policies-ondemand-create.txt]: thanks updated!

File: docs/atlascli/command/atlas-backups-compliancePolicy-policies-ondemand-update.txt (hunk 2 above)
Comment (blva) [reply docs/atlascli/command/atlas-backups-compliancePolicy-policies-ondemand-update.txt]: yep, updated! thanks

**Comment by blva** [comment]
@kanchana-mongodb rfal, I've updated everything that is not --projectId (global flag), --help description or --watch, which are common flags and changing them would change docs everywhere, so better to address separately if we have strong opinion.

**Review by kanchana-mongodb — APPROVED** [review]
LGTM

**Pull request merged**